### ISSUE TITLE
hepatic megalocytosis

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3118]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

hepatic megalocytosis

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

A condition in which hepatocytes exhibit gross enlargement of cytoplasmic and nuclear volume without the formation of mitotic spindles and mitosis, potentially indicating irreversible injury. 

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:11509115

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal hepatocyte morphology (MP:0000607)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:sjwang

**Comment by anna-anagnostop** [comment]
Added new term:
hepatic megalocytosis MP:0031022
